src/chrome/utils.ts

```diff
@@ -13,3 +15,62 @@ export const getCurrentTabUId = (callback: (url: number | undefined) => void): v
         callback(tabs[0].id);
     });
 }
+
+// Encrpts a string using the AES algorithm. Optional parameter: Password, AES Mode
+export function encryptText(text: string, password?: string, mode?: forge.cipher.Algorithm){
+    // password is used to generate key, if none gen random
+    if (password === undefined) {
+    password = forge.random.getBytesSync(48).toString();
+    }
+    
+    // if not mode is specified, use AES-GCM
+    if(mode === undefined) {
+        mode = "AES-GCM"
+    }
+
+    // generate key from password
+    let salt = forge.random.getBytesSync(128);
+    let key = forge.pkcs5.pbkdf2(password, salt, 10000, 32);
+
+    // set IV
+    let iv = forge.random.getBytesSync(16);
+
+    // Encrypt the text
+    let cipher = forge.cipher.createCipher(mode, key);
+    cipher.start({ iv: iv });
+    cipher.update(forge.util.createBuffer(text));
+    cipher.finish();
+
+    let tag = cipher.mode.tag;
+    let cTXT = forge.util.encode64(cipher.output.data);
+
+    return {CipherTXT: cTXT, IV: iv, Key: key, Tag: tag}
```

Comment:
Might create a wrapper function for this depending on how we want to handle key generation/storage and when we want to return the wrapCipherTXT